pulp_2to3_migration/app/plugin/__init__.py

```diff
@@ -1,8 +1,22 @@
+from gettext import gettext as _
+from collections import namedtuple
+import logging
 import pkg_resources
```

Comment:
Stylistic nit-pick. I think throughout our codebase we try to put module import first and import specific items after + everything in lexical order. So it would be something like:
```

￼import logging
￼import pkg_resources

￼from collections import namedtuple
from gettext import gettext as _

```